react/queries/productSearchV3.gql

```diff
@@ -87,5 +87,8 @@ query productSearchV3(
     operator
     searchState
     redirect
+    options {
+      deliveryPromisesEnabled
```

Comment:
⛏️  I generally prefer to name boolean fields with either "is" or "has".   Like isDeliveryPromisesEnabled